### DISCUSSION TITLE
Input widgets error when I configure type arguments

### DISCUSSION THREAD

**Discussion by alainarri** [Textualize/textual#4137] [Q&A]
Input( type ="text",id="txtNpulses")

It will be helpful if you run the following command and paste the results:

```
TypeError: Input.__init__() got an unexpected keyword argument 'type'
```

**Comment by davep** [comment]
What version of Textual are you using? `type` (along with `max_length` and `valid_empty`) was added in 0.42.0, in late November last year.

**Reply by davep** [reply]
@alainarri Did you still need help with this?
